tests/components/formfield.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import FormField from "@libs/components/formfield";
+
+describe("components/FormField", () => {
+  it("can have children", () => {
+    const component = renderer.create(<FormField children="Children is here!" />).toJSON();
```

Comment:
That's not how we pass the `children` prop in React.

Reply:
Also, technically it is not a `component` but a `tree`. If you look at the tests I wrote, I make this distinction.

We don't earn anything by removing intermediate variables, expect less readable code 😉